core/src/main/java/me/mykindos/betterpvp/core/gamer/listeners/GamerListener.java

```diff
@@ -29,13 +39,29 @@ public class GamerListener implements Listener {
     @Config(path="gamer.default.fragments", defaultValue = "0")
     private int defaultFragments;
 
+    @Inject
+    @Config(path = "tab.shop", defaultValue = "mineplex.com/shop")
+    private String shop;
+
+    @Inject
+    @Config(path = "tab.server", defaultValue = "Clans-1")
+    private String server;
+
     private final Core core;
     private final GamerManager gamerManager;
 
     @Inject
     public GamerListener(Core core, GamerManager gamerManager){
         this.core = core;
         this.gamerManager = gamerManager;
+
+        this.header = new PermanentComponent(gamer -> Component.text("Mineplex ", NamedTextColor.GOLD)
+                .append(Component.text("Network ", NamedTextColor.WHITE))
+                .append(Component.text(Objects.requireNonNull(server, ""), NamedTextColor.GREEN)));
+
+        this.footer = new PermanentComponent(gamer -> Component.text("Visit ", NamedTextColor.WHITE)
+                .append(Component.text(Objects.requireNonNull(shop, ""), NamedTextColor.YELLOW))
+                .append(Component.text(" for cool perks!", NamedTextColor.WHITE)));
```

Comment:
Do we actually change this from player to player? Seems pointless storing something thats the same for everybody in gamer